build/Xenko.build

```diff
@@ -117,18 +119,17 @@ Example of use:
       <Output TaskParameter="NuGetVersion" PropertyName="XenkoPackageVersion"/>
     </GenerateVersionFile>
 
-	<Message Text="Done preparing build for version $(XenkoPackageVersion)"/>
-	<TeamCitySetBuildNumber BuildNumber="$(XenkoPackageVersion)" Condition="'$(TEAMCITY_BUILD_PROPERTIES_FILE)' != '' " />
+    <Message Text="Done preparing build for version $(XenkoPackageVersion)"/>
+    <TeamCitySetBuildNumber BuildNumber="$(XenkoPackageVersion)" Condition="'$(TEAMCITY_BUILD_PROPERTIES_FILE)' != '' " />
   </Target>
-  <Target Name="Package" DependsOnTargets="PreparePackageBuild;PrerequisitesInstaller;Build;VSIXPlugin"/>
+  <PropertyGroup>
+    <_XenkoPackageTargetPrerequisitesInstaller Condition="'$(XenkoBuildPrerequisitesInstaller)' == 'true'">PrerequisitesInstaller</_XenkoPackageTargetPrerequisitesInstaller>
+  </PropertyGroup>
+  <Target Name="Package" DependsOnTargets="PreparePackageBuild;$(_XenkoPackageTargetPrerequisitesInstaller);Build;VSIXPlugin"/>
   
-  <Target Name="CopyXenkoKey">
-    <Copy Condition="'$(XenkoSign)' == 'true' And Exists('$(LOCALAPPDATA)\Xenko\XenkoBuildKey\xenko.snk')" SourceFiles="$(LOCALAPPDATA)\Xenko\XenkoBuildKey\xenko.snk" DestinationFolder="$(MSBuildThisFileDirectory)"/>
```

Comment:
Was removing this line intentional? I think this is causing the TeamCity builds to break since it no longer finds the .snk file.

Reply:
We don't .NET sign assembly anymore since a few years ago. It was preventing user to mix with custom assemblies (different identity) and forcing everything to be signed.

Reply:
@Basewq if you are interested, we are currently using these TeamCity settings to build the packages and it works fine:
https://github.com/vvvv/xenko-teamcity

Reply:
Thanks for the info.
I didn't really do a deep investigation, I only noticed the official builds shows everything failing now, with the error:
```
Compilation error: sources\core\Xenko.Core.MicroThreading\Xenko.Core.MicroThreading.csproj
CSC error CS7027: Error signing output with public key from file 'D:\BuildAgent\work\8cafb933a54b01d0\build\xenko.snk' -- Could not find file 'D:\BuildAgent\work\8cafb933a54b01d0\build\xenko.snk'.
```

Reply:
you are right! the codesign step should probably be switched off altogether. @xen2 is there a flag for that? should it be removed from the build step?

Reply:
Thanks for the heads up.
I have removed all the obsolete .NET assembly signing steps in latest master.
I will check if next build will pass or further fixes necessary.

Reply:
Unfortunately, it seems to have another error now...

Reply:
Thanks for the second heads up.
I'll look into it ASAP tomorrow.

Reply:
Yep, there's some signing stuff for Bullet in `Xenko.Physics.csproj`.
FYI, this isn't a priority issue for me, I just noticed the build failing and thought you'd like to know!